### PR TITLE
Modify the Sermant Backend user manual and compatibility documentation

### LINKED ISSUE #342: Modify the Sermant Backend user manual and compatibility documentation.
### What would you like to be added?

1. Modified the configuration management user guide.
2. Added the user manual of the hot swap service.
3. Added the compatibility information about the upgrade to 2.1.0.

### Why is this needed?

The Sermant Backend function is adjusted in Sermant 2.1.0 and later versions.

### PR DESCRIPTION
**Pull request by hanbingleixue**
What type of PR is this?

Doc

What this PR does / why we need it?

1.Modified the configuration management user guide.
2.Added the user manual of the hot swap service.
3.Added the compatibility information about the upgrade to 2.1.0.

Which issue(s) this PR fixes:

Fixes #342

### CODE REVIEW

File: docs/zh/document/user-guide/sermant-backend.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Sermant Backend参数可在编译打包前通过`sermant-backend/src/main/resour
 | dynamic.config.userName    |授权认证时使用的用户名（明文） | null     |    开启授权认证时必须       |
 | dynamic.config.password    |授权认证时使用的密码（采用AES加密后的密文）| null     |    开启授权认证时必须       |
 | dynamic.config.secretKey    |密码采用AES方式加密时使用的密钥 | null     |    开启授权认证时必须       |
+| dynamic.config.template.path   |配置页面的UI模板路径 | null     |    否       |
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-backend.md]: 介绍不够详细。模板怎么写需要有文档示例跳转

File: docs/zh/document/user-guide/sermant-backend.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Sermant Backend使用手册
 
-Sermant Backend包含Sermant数据处理后端模块和前端信息展示模块，旨在为Sermant提供运行时的可观测能力，当前主要包括Sermant Agent心跳信息, 上报事件的接收和展示，webhook推送，配置管理等功能。
+Sermant Backend包含Sermant数据处理后端模块和前端信息展示模块，旨在为Sermant提供运行时的管理能力和可观测能力，当前主要包括Sermant Agent心跳信息, 上报事件的接收和展示，webhook推送，配置管理、热插拔服务等功能。
 
-Sermant Backend与Sermant Agent配合使用。Sermant Agent挂载在宿主应用启动后作为数据发送端，可定时发送当前Sermant Agent的心跳数据(服务名、主机名、实例ID、版本号、IP、时间戳、挂载插件信息)，事件数据(Sermant Agent启停、核心服务启停、字节码增强、日志数据等)。Backend作为数据接收端，可接收处理Sermant Agent发送的心跳和事件等数据，将紧急事件推送至webhook，并在前端可视化展示，提供观测Sermant Agent运行状态的能力。
+Sermant Backend与Sermant Agent配合使用。Sermant Agent挂载在宿主应用启动后作为数据发送端，可定时发送当前Sermant Agent的心跳数据(服务名、主机名、实例ID、版本号、IP、时间戳、挂载插件信息)，事件数据(Sermant Agent启停、核心服务启停、字节码增强、日志数据等)。Backend作为数据接收端，可接收处理Sermant Agent发送的心跳和事件等数据，将紧急事件推送至webhook，并在前端可视化展示，提供观测Sermant Agent运行状态的能力。Sermant Agent挂载在宿主应用启动后还可以作为热插拔功能的服务端，会接收Sermant Backend的热插拔指令，执行插件安装、卸载和升级操作。
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-backend.md]: 功能多了按点列出来，写一起不直观

File: docs/zh/document/user-guide/sermant-backend.md [hunk 3]
```diff
@@ -94,6 +95,10 @@ Sermant Backend使用JDK 1.8版本开发，因此运行环境需JDK 1.8及以上
 agent.service.heartbeat.enable=true
 # Gateway service switch
 agent.service.gateway.enable=true
+# Dynamic config service switch
+agent.service.dynamic.config.enable=true
+# dynamic mount service switch
+agent.service.hot.plugging.service.enable=true
 # Event switch
 event.enable=true
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-backend.md:119]: 各个配置对应功能建议描述下

File: docs/zh/document/user-guide/sermant-backend.md [hunk 4]
```diff
@@ -266,3 +277,57 @@ java -Dserver.port=8989 -javaagent:${path}/sermant-agent/agent/sermant-agent.jar
...
+
+<MyImage src="/docs-img/backend/zh/plugin-install-2.png"></MyImage>
+
+- 选择`安装插件`服务，输入插件名称即可点击`确认`按钮进行插件安装
+
+<MyImage src="/docs-img/backend/zh/plugin-install-3.png"></MyImage>
+
+> 注：动态安装插件前请确保插件的JAR包在${path}/sermant-agent-x.x.x/pluginPackage/${pluginName}下。 ${path}为sermant实际安装路径，x.x.x代表sermant某个版本号，${pluginName}为插件名称
+
+- 插件安装完成之后，即可在实例状态页面的实例信息中看到已经安装的插件信息
```
Comment (lilai23) [inline docs/zh/document/user-guide/sermant-backend.md]: 这里的即可和心跳上报时间间隔有关，建议说明下，否则可能不能立即查看到结果

File: docs/zh/document/user-guide/sermant-backend.md (hunk 1 above)
Comment (hanbingleixue) [reply docs/zh/document/user-guide/sermant-backend.md]: done

File: docs/zh/document/user-guide/sermant-backend.md (hunk 2 above)
Comment (hanbingleixue) [reply docs/zh/document/user-guide/sermant-backend.md]: done

File: docs/zh/document/user-guide/sermant-backend.md (hunk 3 above)
Comment (hanbingleixue) [reply docs/zh/document/user-guide/sermant-backend.md:119]: 上面的参数配置里面已经介绍了

File: docs/zh/document/user-guide/sermant-backend.md (hunk 4 above)
Comment (hanbingleixue) [reply docs/zh/document/user-guide/sermant-backend.md]: done

**Review by lilai23 — APPROVED** [review]

**Pull request merged**